files/default/start-container.sh

```diff
@@ -1,10 +1,12 @@
 #!/bin/bash
 while IFS=, read -r port password ; do
   if [ "$2" == "$port" ] ; then
-    docker run -p "${port}":22 -h dobc --rm -e DOBC_PASSWORD="${password}" \
+    docker run \
+      -p "330${port}":22 \
+      -p "340${port}":8080 \
+      -h dobc --rm -e DOBC_PASSWORD="${password}" \
       --name=dobc-"${port}" -d \
       --device-write-bps /dev/vda:20mb \
-      -c 10 -m 128m \
       osuosl/dobc-centos
```

Comment:
Is it possible to create a "utility" bash file which only includes functions, such as the function to create a container? It would remove the duplicated container creation code.

Something like:
```bash
#!/bin/bash
dobc-container()
{
  port=$1
  password=$2
  docker run \
    -p "330${port}":22 \
    -p "340${port}":8080 \
    -h dobc --rm -e DOBC_PASSWORD="${password}" \
    --name=dobc-"${port}" -d \
    --device-write-bps /dev/sda:20mb \
    osuosl/dobc-centos
}
```

Which can be included in the `start-container.sh` and `start-all.sh` files with a `. utility.sh` command and called with: `dobc-container <port-ending> <password>`

For example, calling `dobc-container 03 apple`, will create a docker container with the port mappings 33003->22 and 34003->8080 with the password apple.